cmd/notary/keys.go

```diff
@@ -361,13 +361,11 @@ func (k *keyCommander) keysRotate(cmd *cobra.Command, args []string) error {
 		rolesToRotate = []string{data.CanonicalSnapshotRole}
 	case data.CanonicalTargetsRole:
 		rolesToRotate = []string{data.CanonicalTargetsRole}
+	case data.CanonicalTimestampRole:
```

Comment:
Filed https://github.com/docker/notary/issues/554 for it.  Basically am trying to keep this PR under 1k lines :D